cli/internal/helm/loader.go

```diff
@@ -134,14 +134,48 @@ func (i *ChartLoader) loadConstellationServices(csp cloudprovider.Provider,
 			"image":        versions.JoinImage,
 			"namespace":    constants.ConstellationNamespace,
 		},
+		"ccm": map[string]interface{}{
+			"csp": csp,
+		},
 	}
 
-	if csp == cloudprovider.Azure {
-		joinServiceVals, ok := vals["join-service"].(map[string]any)
-		if !ok {
-			return helm.Release{}, errors.New("invalid join-service values")
+	switch csp {
+	case cloudprovider.Azure:
+		{
+			joinServiceVals, ok := vals["join-service"].(map[string]any)
+			if !ok {
+				return helm.Release{}, errors.New("invalid join-service values")
+			}
+			joinServiceVals["enforceIdKeyDigest"] = enforceIDKeyDigest
+
+			ccmVals := vals["ccm"].(map[string]any)
+			ccmVals["Azure"] = map[string]any{
+				"image": versions.VersionConfigs[k8sVersion].CloudControllerManagerImageAzure,
+			}
```

Comment:
```suggestion
			vals["ccm"].(map[string]any)["Azure"] = map[string]any{
				"image": versions.VersionConfigs[k8sVersion].CloudControllerManagerImageAzure,
			}
```
Same thing as already mentioned previously.
Also fine to leave as is